Frontend/ui-driver/src/Screens/SubscriptionScreen/Controller.purs

```diff
@@ -307,6 +320,8 @@ eval (LoadMyPlans plans) state = do
       case currentPlanResp.mandateDetails of 
         Mb.Nothing -> continue newState
         Mb.Just (MandateData mandateDetails) -> do
+          let payerVpa = (Mb.fromMaybe "" mandateDetails.payerVpa)
+              pspLogo = Const.getPspIcon payerVpa <> "," <> (getAssetStoreLink FunctionCall) <> (Const.getPspIcon payerVpa) <> ".png"
```

Comment:
We can store `Const.getPspIcon payerVpa`  in a var and make function call only once